Packs/ConcentricAI/Playbooks/ConcentricAI_Demo.yml

```diff
@@ -0,0 +1,92 @@
+id: ConcentricAI Demo Playbook
+version: -1
+fromversion: 6.0.0
+name: ConcentricAI Demo Playbook
+description: This playbook will show how to handle a risk incident and fetch all appropriate
+  file details with respect to it.
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 8b07a56d-2681-492d-8109-fe89ddcd70a4
+    type: start
+    task:
+      id: 8b07a56d-2681-492d-8109-fe89ddcd70a4
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "4"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 170
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "4":
+    id: "4"
+    taskid: 5a29c2af-6511-4acf-8e7a-f3c0eaa87859
+    type: regular
+    task:
+      id: 5a29c2af-6511-4acf-8e7a-f3c0eaa87859
+      version: -1
+      name: get-file-details
+      description: Get's file information
+      script: ConcentricAI|||get-file-details
```

Comment:
what's the difference between the both ? they are both commands from our integration itself right ?

Reply:
When we don't bind the command to an integration it can be a lot easier to release a V2 in the future, that way there will be no need to update the playbook to use a specific version but it can choose the one configured.

Reply:
done.